fuzzers/035a-iob-idelay/top.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python3
+
+import os, random
+random.seed(int(os.getenv("SEED"), 16))
+
+import re
+import json
+
+from prjxray import util
+from prjxray.db import Database
+
+# =============================================================================
+
+
+def get_loc(name):
+    m = re.match("^\S+_X([0-9]+)Y([0-9]+)$", name)
```

Comment:
I believe ``get_loc`` can be replaced by a call to [util.create_xy_fun](https://github.com/SymbiFlow/prjxray/blob/master/fuzzers/040-clk-hrow-config/top.py#L11)

Reply:
It can't be used here as it assumes constant prefix but I am searching for LIOB33 and RIOB33 which differ.

Reply:
Wouldn't ``util.create_xy_fun('[LR]IOB33')`` work?

Reply:
Right, that works :)